tests/test_tokens.py

```diff
@@ -88,17 +88,6 @@ def setUp(self, *args, **kwargs):
         self.reg = AbstractNode.objects.get(Q(**{self.Model.SHORT_NAME: self.sanction}))
         self.user = self.reg.creator
 
-    @pytest.mark.usefixtures('mock_gravy_valet_get_verified_links')
-    def test_sanction_handler(self):
-        if not self.kind:
-            return
-        approval_token = self.sanction.approval_state[self.user._id]['approval_token']
-        handler = TokenHandler.from_string(approval_token)
-        with mock_auth(self.user):
-            with mock.patch(f'osf.utils.tokens.handlers.{self.kind}_handler') as mock_handler:
-                handler.to_response()
-                mock_handler.assert_called_with('approve', self.reg, self.reg.registered_from)
-
```

Comment:
I see. Now I remember, this is the one that causes conflict. The latest B&I work `develop` removed but we added it back by accident.